container/docker-entrypoint-prod.sh

```
@@ -0,0 +1,46 @@
+export STORAGE_DIR="${STORAGE_DIR:-/app/data}"
+export DATA_DIR="${DATA_DIR:-/app/data}"
+export WORKING_DIR="${WORKING_DIR:-/app}"
+export CONF_DIR="${CONF_DIR:-/app/conf}"
+
+# Ensure essential directories exist
+mkdir -p "$DATA_DIR" "$CONF_DIR" /app/staticfiles
+
+# Validate critical environment variables
+if [ -z "$DJANGO_SECRET_KEY" ]; then
+    echo "⚠️ WARNING: DJANGO_SECRET_KEY not set. Using temporary key."
+    export DJANGO_SECRET_KEY="prod-temp-key-$(date +%s)"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)